addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -68,6 +68,28 @@ def _domain_employee_id(self):
     partner_id = fields.Many2one(compute='_compute_partner_id', store=True, readonly=False)
     readonly_timesheet = fields.Boolean(compute="_compute_readonly_timesheet", compute_sudo=True, export_string_translation=False)
     milestone_id = fields.Many2one('project.milestone', related='task_id.milestone_id')
+    message_partner_ids = fields.Many2many('res.partner', compute='_compute_message_partner_ids',
+                                           search='_search_message_partner_ids', readonly=True)
```

Comment:
By default, a computed field is not stored and readonly. 🙂 
```suggestion
                                           search='_search_message_partner_ids')
```

Reply:
fixed